src/Umbraco.Web/Search/UmbracoTreeSearcher.cs

```diff
@@ -85,6 +85,7 @@ public class UmbracoTreeSearcher
                     break;
                 case UmbracoEntityTypes.Media:
                     type = "media";
+                    fields = new[] { "id", "__NodeId", UmbracoExamineIndex.UmbracoFileFieldName};
```

Comment:
Should we just make the above `fields` definition a List, so that you can just add fields you want to it instead of re-creating it, just means less duplicated code (same goes for the member stuff above)

Reply:
Yes, would love to do that